### ISSUE TITLE
Unknown column 'LastAccessedSite' error in query

### ISSUE THREAD

**Issue by minetoblend** [apollo-dw/omdb#90]
I just did the basic setup to run the server locally and I'm getting errors now that a query is accessing an unknown column. Pretty sure that that the init.sql file is no longer up to date with the expected database schema.

![image](https://github.com/apollo-dw/omdb/assets/8039761/c21eba4d-8ee6-49e8-b571-11edf8cf1391)

**Comment by apollo-dw** [comment]
Yeah sorry, the `init.sql` is extremely out of date. I can look into updating it within the next few days if I have the time

**Comment by VINXIS** [comment]
i think u updated it
